tools/wpt/utils.py

```diff
@@ -55,9 +55,19 @@ def unzip(fileobj, dest=None, limit=None):
         for info in zip_data.infolist():
             if limit is not None and info.filename not in limit:
                 continue
-            zip_data.extract(info, path=dest)
-            perm = info.external_attr >> 16 & 0x1FF
-            os.chmod(os.path.join(dest, info.filename), perm)
+            info_st_mode = info.external_attr >> 16
+            info_dst_path = os.path.join(dest, info.filename)
+            if stat.S_ISLNK(info_st_mode):
+                link_src_path = zip_data.read(info)
```

Comment:
Can you put a comment in about the representation of links in zip files (like the commit message).